client/my-sites/plans-features-main/components/plan-upsell-modal/components/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { Button } from '@automattic/components';
+import styled from '@emotion/styled';
+
+export const DialogContainer = styled.div`
+	padding: 24px;
+`;
+
+export const Heading = styled.div`
+	font-family: Recoleta;
+	color: var( --studio-gray-100 );
+	font-size: 22px;
+	line-height: 26px;
+	letter-spacing: 0.38px;
```

Comment:
`Non-blocking / nitpick`: 

I totally understand that this PR is simply moving components around, and it doesn't have to be addressed in these changes, but I'm seeing what seem like non-standardized properties. From brief searching, it looks like letter-spacing ( `letter-spacing: -0.32px;`, `letter-spacing: -0.15px;` ) and font-size ( `font-size: 22px;` ) seem to be pretty specific to our work.

Curious if you could elaborate on this more @jdc91